code/modules/antagonists/bloodsuckers/powers/gangrel.dm

```diff
@@ -15,125 +19,64 @@
-	..()
 
 /datum/action/bloodsucker/gangrel/transform/ActivatePower()
 	var/datum/antagonist/bloodsucker/bloodsuckerdatum = owner.mind.has_antag_datum(/datum/antagonist/bloodsucker)
 	var/mob/living/carbon/human/user = owner
 	var/datum/species/user_species = user.dna.species
+	var/minortransformdone = FALSE
+	var/mediumtransformdone = FALSE
 	user.Immobilize(10 SECONDS)
 	if(!do_mob(user, user, 10 SECONDS, 1))
 		return
 	switch(bloodsuckerdatum.total_blood_drank)
 		if(0 to 500)
-			if(iscatperson(user))
-				user.set_species(/datum/species/lizard)
-				playsound(user.loc, 'sound/voice/lizard/hiss.ogg', 50)
-			else
-				user.set_species(/datum/species/human/felinid)
-				playsound(user.loc, 'sound/voice/feline/meow1.ogg', 50)
+			if(!minortransformdone)
+				if(iscatperson(user))
+					user.set_species(/datum/species/lizard)
+					playsound(user.loc, 'sound/voice/lizard/hiss.ogg', 50)
+				else
+					user.set_species(/datum/species/human/felinid)
+					playsound(user.loc, 'sound/voice/feline/meow1.ogg', 50)
+					if(DIGITIGRADE in user_species.species_traits)
+						user_species.species_traits -= DIGITIGRADE
+				minortransformdone = TRUE
+				user_species.punchdamagelow += 5.0
+				user_species.punchdamagehigh += 5.0 //stronk
+				user_species.armor += 30
+				to_chat(user, span_notice("You aren't strong enough to morph into something stronger! But you do certainly feel more feral and stronger than before."))
+				return
+			if(minortransformdone)
+				user_species.punchdamagelow += 1
+				user_species.punchdamagehigh += 1
+				user_species.armor += 2.5
+				to_chat(user, span_notice("Using a bit more of blood you manage to get a tad stronger, but you still haven't evolved your ability yet."))
+				bloodsuckerdatum.AddBloodVolume(50)
+		if(500 to 1000)
+			if(!mediumtransformdone)
+				user.set_species(/datum/species/gorilla)
+				playsound(user.loc, 'sound/creatures/gorilla.ogg', 50)
 				if(DIGITIGRADE in user_species.species_traits)
 					user_species.species_traits -= DIGITIGRADE
-			user_species.punchdamagehigh += 5.0 //stronk
-			user_species.armor += 30
-			to_chat(user, span_notice("You aren't strong enough to morph into something stronger! But you do certainly feel more feral and stronger than before."))
-		if(500 to 1000)
-			user.set_species(/datum/species/gorilla)
-			to_chat(owner, span_notice("You transform into a gorrila-ey beast, you feel stronger!"))
-			playsound(user.loc, 'sound/creatures/gorilla.ogg', 50)
-			if(DIGITIGRADE in user_species.species_traits)
-				user_species.species_traits -= DIGITIGRADE
-			user_species.punchdamagehigh += 7.5 //very stronk
-			user_species.armor += 35
+				mediumtransformdone = TRUE
+				user_species.punchdamagelow += 7.5
+				user_species.punchdamagehigh += 7.5 //very stronk
+				user_species.armor += 35
+				to_chat(owner, span_notice("You transform into a gorrila-ey beast, you feel stronger!"))
+				return
+			if(mediumtransformdone)
+				user_species.punchdamagelow += 2
```

Comment:
Do something else here if not giving a flat refund I dont think its a good idea to give MASSIVE innate armor and punch damage if it doesnt cost anything else